test/puppetlabs/trapperkeeper/services_test.clj

```diff
@@ -62,6 +63,55 @@
           app (bootstrap-services-with-empty-config [service1])]
       (is (not (nil? app))))))
 
+(deftest test-restart-file-correct-count
+  (testing "create a restart file and check that it correctly increments when a service starts"
+    (let [service1 (service Service1
+                            []
+                            (service1-fn [this] "hi"))
+          service2 (service Service1
+                            []
+                            (service1-fn [this] "hi"))]
+          (with-app-with-config app [service1] {:global {:restart-file "./dev-resources/counter"}}
+            (= (slurp "./dev-resources/counter") "1"))
```

Comment:
For tests, you'll want to use the `is` function around any of your equality tests to ensure that you are asserting equality and not just calling the `=` function.